packages/component-library/src/components/form/mt-number-field/mt-number-field.vue

```diff
@@ -398,6 +408,12 @@ export default defineComponent({
     }
   }
 }
+
+.mt-number-field {
+  input.mt-number-field__align-end {
+    text-align: end;
+  }
+}
```

Comment:
I guess it makes sense to only use `.mt-number-field__align-end` as the selector as the extra scoping to `.mt-number-field` is unnecessary. It's already included in the longer selector.